src/app/register/pages/register/register.component.ts

```diff
@@ -152,9 +155,11 @@ export class RegisterComponent implements OnInit, AfterViewInit {
           if (response.url) {
             this._gtag
               .reportEvent(
-                'RegGrowth',
                 'New-Registration',
-                this.requestInfoForm || 'Website'
+                'RegGrowth',
```

Comment:
Great catch! the ReGrowth is a category NOT an action. 
Now this will break my wrong tests here:

https://github.com/ORCID/orcid-angular/blob/88ab2416b78ba3a6830a6042798c7eda3430dc17/cypress/support/google-analytics.commands.js#L103-L105

So I will add one or two commits to this PR to fix it